cmd/tfinstall/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/hashicorp/logutils"
+	"github.com/hashicorp/terraform-exec/tfinstall"
+	"github.com/mitchellh/cli"
+)
+
+func main() {
+	filter := &logutils.LevelFilter{
+		Levels:   []logutils.LogLevel{"DEBUG", "WARN", "ERROR"},
+		MinLevel: logutils.LogLevel("WARN"),
+		Writer:   os.Stderr,
+	}
+	log.SetOutput(filter)
+
+	ui := &cli.ColoredUi{
+		OutputColor: cli.UiColorBlue,
+		InfoColor:   cli.UiColorGreen,
```

Comment:
Nitpick: Red for errors and yellow for warnings is common, but is there any particular reason not to keep `Output` and `Info` as default (probably white)? I think that too many colours then defeat the purpose of colouring (catching attention) and errors and warnings get lost.

Reply:
Thanks, that does make more sense here - I copied these from tf-sdk-migrator, which has more complex output.